### PR TITLE
Added BSC Network

### PR DESCRIPTION
**Pull request by acharyarupak391**
- Added BSC Network:
  - protocol contracts pages
  - homepage Launch App popup
  - marketplace page
- Upated builder to fetch bsc protocol contracts

### CODE REVIEW

File: src/views/marketplace/networks.ts [hunk 1]
```diff
@@ -28,6 +28,17 @@ const networks: NetworkOption[] = [
     ).then((x: any) => x.default),
     text: 'arbitrum.neptunemutual.net',
     link: 'https://arbitrum.neptunemutual.net'
+  },
+  {
+    name: 'BSC (BNB Smart Chain)',
```
Comment (binodnpm) [inline src/views/marketplace/networks.ts:33]: BNB Smart Chain

File: src/views/homepage/launch-app/networks.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    link: 'arbitrum.neptunemutual.net'
+  },
+  {
+    value: 'Ethereum',
+    id: 'ethereum',
+    href: 'https://ethereum.neptunemutual.net',
+    link: 'ethereum.neptunemutual.net'
+  },
+  {
+    value: 'BSC (BNB Smart Chain)',
```
Comment (binodnpm) [inline src/views/homepage/launch-app/networks.ts:20]: BNB Smart Chain

File: src/views/marketplace/networks.ts (hunk 1 above)
Comment (rupak-nm) [reply src/views/marketplace/networks.ts]: Updated

File: src/views/homepage/launch-app/networks.ts (hunk 2 above)
Comment (rupak-nm) [reply src/views/homepage/launch-app/networks.ts]: Updated

**Review by binodnpm — APPROVED** [review]

**Pull request merged**